### PR TITLE
Make testing simple

### PR DESCRIPTION
**Pull request by osamingo**
The title said all.

### CODE REVIEW

**Comment by osamingo** [comment]
@YoshiyukiMineo Please review it if you have free time :)

File: sonyflake_test.go [hunk 1]
```diff
@@ -139,16 +137,17 @@ func TestSonyflakeInParallel(t *testing.T) {
...
-	set := mapset.NewSet()
+	set := make(map[uint64]struct{})
 	for i := 0; i < numID*numGenerator; i++ {
 		id := <-consumer
-		if set.Contains(id) {
-			t.Fatal("duplicated id")
-		} else {
-			set.Add(id)
+		for k := range set {
+			if k == id {
```
Comment (YoshiyukiMineo) [inline sonyflake_test.go:144]: I think it's better to use  the “comma ok” idiom instead of the for loop:
```
if _, ok := set[id]; ok {
...
}
```

Comment (osamingo) [reply sonyflake_test.go]: @YoshiyukiMineo I fixed it. PTAL 🙇

**Comment by YoshiyukiMineo** [comment]
Thank you.

**Pull request merged**